pkg/diagnostics/resiliency_monitoring_test.go

```diff
@@ -29,6 +29,19 @@ const (
 	testStateStoreName           = "testStateStore"
 )
 
+var latencyDistribution = view.Distribution(5, 50, 500, 5_000)
+
+// TODO(jfreeland): Remove.  Troubleshooting only.
+// type customMetricsExporter struct{}
+//
+// func (ce *customMetricsExporter) ExportView(vd *view.Data) {
+// 	log.Printf("vd.View: %+v\n%v\n", vd.View, vd.Rows)
+// 	for i, row := range vd.Rows {
+// 		log.Printf("\tRow: %d: %v\n", i, row)
+// 	}
+// 	log.Printf("StartTime: %s EndTime: %s\n\n", vd.Start.Round(0), vd.End.Round(0))
+// }
+
```

Comment:
So I added this block of code and the `TODO` a couple lines below it in both this PR and in `master` and what I'm seeing is that latency views are simply not getting registered on `master`, while they are attempting to get registered repeatedly in this branch.  I have to clean them up in tests, https://github.com/jfreeland/dapr/pull/1, in order to not stomp on the namespace.